lighty-modules/lighty-jetty-server/src/main/java/io/lighty/server/Http2LightyServerBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright (c) 2022 PANTHEON.tech s.r.o. All Rights Reserved.
+
+   This program and the accompanying materials are made available under the
+   terms of the Eclipse Public License v1.0 which accompanies this distribution,
+   and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+package io.lighty.server;
+
+import io.lighty.server.config.SecurityConfig;
+import java.net.InetSocketAddress;
+import org.eclipse.jetty.alpn.server.ALPNServerConnectionFactory;
+import org.eclipse.jetty.http.HttpVersion;
+import org.eclipse.jetty.http2.server.HTTP2ServerConnectionFactory;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.SecureRequestCustomizer;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+
+public class Http2LightyServerBuilder  extends LightyServerBuilder {
```

Comment:
Change `Http2LightyServerBuilder  extends` to `Http2LightyServerBuilder extends`.

Reply:
done